src/managers/message-manager.js

```diff
@@ -264,14 +265,19 @@ async function handleGistEvents(e) {
         break;
       }
       case "sizeChanged": {
-        log(`Size Changed Width: ${e.data.gist.parameters.width} - Height: ${e.data.gist.parameters.height}`)
+        log(`Size Changed Width: ${e.data.gist.parameters.width} - Height: ${e.data.gist.parameters.height}`);
```

Comment:
This is probably not for now, but it would be nice to replace all those levels of indirection on e.data.gist with something like: 

const { action, name, width, height, title } = e.data.gist.parameters;

-----

There might be room to do: 
e.data.gist.parameters?.key since at the top you are checking the e.data.gist existence already